gensim/models/hdpmodel.py

```diff
@@ -80,65 +122,125 @@ def lda_e_step(doc_word_ids, doc_word_counts, alpha, beta, max_iter=100):
+        Number of documents in the corpus.
+    m_Elogbeta : numpy.ndarray:
+        Stores value of dirichlet expectationn, i.e., compute :math:`E[log \\theta]` for a vector
+        :math:`\\theta \sim Dir(\\alpha)`.
+    m_lambda : {numpy.ndarray, float}
+        Drawn samples from the parameterized gamma distribution.
+    m_lambda_sum : {numpy.ndarray, float}
+        An array with the same shape as `m_lambda`, with the specified axis (1) removed.
+    m_num_docs_processed : int
+        Number of documents finished processing.This is incremented in size of chunks.
+    m_r : list
+        Acts as normaliser in lazy updating of `m_lambda` attribute.
+    m_rhot : float
+        Assigns weight to the information obtained from the mini-chunk and its value it between 0 and 1.
+    m_status_up_to_date : bool
+        Flag to indicate whether `lambda `and :math:`E[log \\theta]` have been updated if True, otherwise - not.
+    m_timestamp : numpy.ndarray
+        Helps to keep track and perform lazy updates on lambda.
+    m_updatect : int
+        Keeps track of current time and is incremented everytime :meth:`~gensim.models.hdpmodel.HdpModel.update_lambda`
+        is called.
+    m_var_sticks : numpy.ndarray
+        Array of values for stick.
+    m_varphi_ss : numpy.ndarray
+        Used to update top level sticks.
+    m_W : int
+        Length of dictionary for the input corpus.
 
     """
-
     def __init__(self, corpus, id2word, max_chunks=None, max_time=None,
                  chunksize=256, kappa=1.0, tau=64.0, K=15, T=150, alpha=1,
                  gamma=1, eta=0.01, scale=1.0, var_converge=0.0001,
                  outputdir=None, random_state=None):
         """
-        `gamma`: first level concentration
-        `alpha`: second level concentration
-        `eta`: the topic Dirichlet
-        `T`: top level truncation level
-        `K`: second level truncation level
-        `kappa`: learning rate
-        `tau`: slow down parameter
-        `max_time`: stop training after this many seconds
-        `max_chunks`: stop after having processed this many chunks (wrap around
-        corpus beginning in another corpus pass, if there are not enough chunks
-        in the corpus)
+        Parameters
+        ----------
+        corpus : iterable of list of (int, float)
+            Corpus in BoW format.
+        id2word : :class:`~gensim.corpora.dictionary.Dictionary`
+            Dictionary for the input corpus.
+        max_chunks : int, optional
+            Upper bound on how many chunks to process. It wraps around corpus beginning in another corpus pass,
+            if there are not enough chunks in the corpusю
+        max_time : int, optional
+            Upper bound on time (in seconds) for which model will be trained.
+        chunksize : int, optional
+            Number of documents in one chunck
+        kappa : float, optional
+            Learning rate
+        tau : float, optional
+            Slow down parameter
```

Comment:
What does this mean, can you describe it in more details? If something isn't clear - this is a bad description.

Reply:
is this fine -
```
kappa: float,optional
 Learning parameter which acts as exponential decay factor to influence extent of learning from each batch.
tau: float, optional
  Learning parameter which down-weights early iterations of documents.```

Reply:
@gyanesh-m sounds better than current description :+1: